src/libfetchers/path.cc

```diff
@@ -140,6 +141,20 @@ struct PathInputScheme : InputScheme
                 mtime = dumpPathAndGetMtime(absPath.string(), sink, defaultPathFilter);
             });
             storePath = store->addToStoreFromDump(*src, "source");
+        } else {
+            // To avoid copying the path again to the /nix/store, we need to add a cache entry.
+            ContentAddressMethod method = ContentAddressMethod::Raw::NixArchive;
+            auto fp = getFingerprint(store, input);
+            if (fp) {
+              auto cacheKey = fetchers::Cache::Key{"fetchToStore", {
```

Comment:
Indentation is off here.

Reply:
Do you know where I could get "path" from this this case? Than I can easily move this into a function.

Reply:
Like this?